src/Commands/Configure.ts

```diff
@@ -44,6 +44,13 @@ export class Configure implements CommandModule {
         boolean: true,
         describe: `Start a configuration wizard without GUI`
       })
+      .option('networkonly', {
+        default: false,
+        boolean: true,
+        alias: 'networkTestOnly',
```

Comment:
The preferred way is to use shorthand aliases. But when the option name is too complex, it will be better to avoid any aliases